sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_tool_call_accuracy/_tool_call_accuracy.py

```diff
@@ -246,97 +239,96 @@ async def _real_call(self, **kwargs):
 
-        Exact implementation might need to vary slightly depending on the results produced.
-        Default behavior is to average the all number-based outputs.
-
-        :param per_turn_results: List of evaluation results for each turn in the conversation.
-        :type per_turn_results: List[Dict]
-        :return: A dictionary containing aggregated results, with numeric metrics having their
-        means as top-level values in the dictionary, and all original
-        values (including non-numerics) located in under the "evaluation_per_turn" key,
-        which each sub-key being a metric and each sub-value being a the list of that metric's
-        per-turn values.
-        :rtype: AggregateResult[T_EvalValue]
+        }
+    
+    def _parse_tools_from_response(self, response):
+        """Parse the response to extract tool calls and results.
+        :param response: The response to parse.
+        :type response: Union[str, List[dict]]
+        :return: List of tool calls extracted from the response.
+        :rtype: List[dict]
         """
-
-        aggregated: Dict[str, Union[float, Dict[str, List[T_EvalValue]]]] = {}
-        evaluation_per_turn: Dict[str, List[T_EvalValue]] = {}
-
-        # Go over each turn, and rotate the results into a
-        # metric: List[values] format for the evals_per_turn dictionary.
-
-        num_evaluated = len([per_turn_result for per_turn_result in per_turn_results
-                             if per_turn_result.get(self._result_key) != self._NOT_APPLICABLE_RESULT])
-        if num_evaluated == 0:
-            # None of the invoked tools were applicable, return not applicable result
-            # (If a tool fails evaluation, we'll throw an exception)
-            return {self._AGGREGATE_RESULT_KEY: self._NOT_APPLICABLE_RESULT,
-                    f"{self._AGGREGATE_RESULT_KEY}_result": self._NOT_APPLICABLE_RESULT,
-                    f"{self._AGGREGATE_RESULT_KEY}_threshold": self.threshold,
-                    f"{self._AGGREGATE_RESULT_KEY}_reason":
-                        "Tool call accuracy evaluation is not yet supported for the invoked tools.",
-                    "per_tool_call_details": []
-                    }
-        # ignore not_applicable results, where the _result_key will be "not applicable"
-        score = sum([per_turn_result.get(self._result_key) == True for per_turn_result in per_turn_results])/num_evaluated
-        aggregated[self._AGGREGATE_RESULT_KEY] = score
-        aggregated[f'{self._AGGREGATE_RESULT_KEY}_result'] = self._PASS_RESULT if score >= self.threshold else self._FAIL_RESULT
-        aggregated[f'{self._AGGREGATE_RESULT_KEY}_threshold'] = self.threshold
-        aggregated["per_tool_call_details"] = per_turn_results
-        return aggregated
+        tool_calls = []
+        tool_results_map = {}
+        if isinstance(response, list):
+            for message in response:                
+                print(message)
```

Comment:
Please remove the print commands. If you would like to log, please use logger with correct log level